src/Microsoft.Azure.WebJobs.Host/WebJobs.Host.csproj

```diff
@@ -29,6 +29,7 @@
   <PackageReference Include="Microsoft.Extensions.Hosting" Version="2.1.0" />
   <PackageReference Include="Microsoft.Extensions.Logging" Version="2.1.0" />
   <PackageReference Include="Microsoft.Extensions.Logging.Configuration" Version="2.1.0" />
+  <PackageReference Include="Polly" Version="7.2.0" />
```

Comment:
How do we feel about this dependency from both the perspective of:
a) the surface area change for WebJobs users - are we planning on releasing this as a minor version bump? 
b) any existing functions users that are already referencing Polly

Reply:
We might consider not taking a dependency on Polly, and instead provide our own simple RetryPolicy type(s), and make it pluggable via DI. Customers can then use what they want.

Reply:
An implementation based on Polly could be broken off into its own package and outside of the SDK. That's the approach taken by the HTTP client factory to avoid this situation.

Reply:
I added WebJobs.Extensions.Polly with the reference to Polly which can be added to an extension via DI.